tests/components/vizio/test_media_player.py

```diff
@@ -107,6 +107,7 @@
         assert len(hass.states.async_entity_ids(MP_DOMAIN)) == 0
 
 
+# pylint: disable=keyword-arg-before-vararg
```

Comment:
Why are we disabling this?

Reply:
It was a byproduct of the last release. You'll notice that args and kwargs are after a keyword argument. I have since updated this argument to a positional argument this so I removed the disable accordingly